src/plone/app/workflowmanager/browser.txt

```diff
@@ -34,7 +34,7 @@ There should not be any plone workflows by default listed.
     False
 
     >>> 'one_state_workflow' in browser.contents
-    False
+    True
```

Comment:
This doesn't look right, given "There should not be any plone workflows by default listed", especially when allow_failures is on for p5.0 and p5.1

Reply:
@jaroel the plone.app.contenttypes test layer enables the one_state_workflow by default. If I recall correctly this was even added in a minor release of p.a.contenttypes.

Reply:
@jaroel here you go: https://github.com/plone/plone.app.contenttypes/blob/1.1.x/plone/app/contenttypes/testing.py#L64

Reply:
Thats a different workflow :D

Reply:
Wait. this is simple_publication_workflow, we have one_state_workflow. I might have been on the wrong track...

Reply:
@jaroel I digged a bit deeper and it turns out this is actually the comment_one_state_workflow which bleeds into the test. I have no idea why plone.app.discussion is enabled in the default test layer here. That shouldn't be the case. I couldn't find any traces of it in plone.app.testing thg.

Reply:
@jaroel turned out to be plone.app.contenttypes after all: https://github.com/plone/plone.app.contenttypes/blob/1.1.x/plone/app/contenttypes/profiles/default/metadata.xml#L11